pkg/operator/deploy/staticresources/master/deployment.yaml

```diff
@@ -27,6 +27,13 @@ spec:
         - master
         image: GENERATED
         name: aro-operator
+        resources:
+          limits:
+            cpu: "1"
```

Comment:
I should've caught this before 🤦🏽 . This looks a little high.  Doing a `oc adm top pod -n openshift-azure-operator` shows consumption less than 1m CPU and memory around 62Mi.  Where as the `openshift-apiserver` pods have around 14-23m CPUs.  This was never profiled but I'd imagine it wouldn't be necessary to dedicate 1 CPU to this deployment.

Reply:
@troy0820 does 100m sound reasonable as a guard rail, or should we go lower?

Reply:
1 CPU really sounds too much. I would say do something like:
```
10m           100Mi            
```
based on:
```
aro-operator-master-7b484568c-lfk24    2m           70Mi            
aro-operator-worker-746c98d467-xkdqs   1m           39Mi 
```

Reply:
The operator checks in the E2E are failing with those limits in place:
```
  Unexpected error:
      <*errors.errorString | 0xc000252060>: {
          s: "timed out waiting for the condition",
      }
      timed out waiting for the condition
  occurred
```
Since the limits/requests were the only change here, I am going to attempt something a little higher (QOS Guaranteed still) and retry E2E.

Reply:
would be good to find out timeout reason. :) nobody told this will be easy

Reply:
Attempted E2E locally, with 100m and 100Mi:
```
NAME                                   READY   STATUS             RESTARTS   AGE
aro-operator-master-69ff655bb6-2jr8k   0/1     CrashLoopBackOff   5          3m13s
aro-operator-worker-7bb49487fd-v6jpp   1/1     Running            0          3m

    lastState:
      terminated:
        exitCode: 137
        reason: OOMKilled
```
Normal runtime during E2E looks like this:
```
NAME                                   CPU(cores)   MEMORY(bytes)   
aro-operator-master-c8c7b9748-nkl6t    16m          104Mi
```
With this info I would go to `50m/500Mi` knowing that we will be adding more controllers to `aro-operator-master`, and since `aro-operator-worker` is only running the Checker controller it can probably stay at `10m/100Mi`.